game/scripts/npc/abilities/death_prophet_silence.txt

```diff
@@ -23,11 +23,11 @@
     // Time
     //-------------------------------------------------------------------------------------------------------------
     "AbilityCooldown"                                     "15 14 13 12 12 12"
-    "AbilityDuration"                                     "3.0 3.5 4.0 4.5 5.0 5.5" //OAA
+    "AbilityDuration"                                     "3.0 4.0 5.0 6.0 6.0 6.0"
 
     // Cost
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityManaCost"                                     "80 90 100 110 165 275"
+    "AbilityManaCost"                                     "80 75 70 65 60 55" //OAA
```

Comment:
I changed my mind. This is fine.

Reply:
I'd still suggest a revert just because this isn't a buff and seems like a random number change as DP does not and should not have mana issues whatsoever. I think the cd buff is good enough to make this ability better and would just suggest making the manacost what it is currently,

Reply:
I'd still suggest a revert just because this isn't a buff and seems like a random number change as DP does not and should not have mana issues whatsoever. I think the cd buff is good enough to make this ability better and would just suggest making the manacost what it is currently,